pkg/controller/virtinformers.go

```diff
@@ -182,6 +185,14 @@ func (f *kubeInformerFactory) ConfigMap() cache.SharedIndexInformer {
 	})
 }
 
+func (f *kubeInformerFactory) LimitRanges() cache.SharedIndexInformer {
+	return f.getInformer("limitrangeInformer", func() cache.SharedIndexInformer {
+		restClient := f.clientSet.CoreV1().RESTClient()
+		lw := cache.NewListWatchFromClient(restClient, "limitranges", k8sv1.NamespaceAll, fields.Everything())
+		return cache.NewSharedIndexInformer(lw, &k8sv1.LimitRangeList{}, f.defaultResync, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
You added a namespace indexer to the informer, which is good. You can make us of the extra index further down.